Test-network.md

```diff
@@ -16,25 +15,27 @@ installed run:
 ```bash
 git clone https://github.com/storj/storj.git storj
 cd storj
-go install storj.io/storj/cmd/{storj-sdk,bootstrap,satellite,storagenode,uplink,gateway}
+make install-sim
 ```
 
 _Ensure that `storj` folder is outside of `GOPATH`, otherwise you may see errors._
 
-This will install the storj-sdk satellite storage node gateway and uplink binaries to wherever Go is configured to output binaries on your system. By default, this is ~/go/bin.
+This will install the storj-sim satellite storage node gateway and uplink binaries to wherever Go is configured to output binaries on your system. By default, this is `~/go/bin`.
 
 
 Next, run setup:
 
 ```bash
-storj-sdk network setup
+storj-sim network setup
 ```
 
 You now have a configured Storj test network with default configuration options.
 
-You might also want to take a look at the config by navigating to the root directory `--config-dir` where all the configs are specified.
+You might also want to take a look at the config by navigating to the root 
+directory `--config-dir` where all the configs are specified.
+You can tweak the configuration settings there as needed.
 
-You can use vim to tweak the default configuration settings. You can also see what is being overwritten on the command-line level with `storj-sdk -x network run`.
+To see what is happening under the hood you can use `-x` or `--print-commands` to see how the processes are exactly started.
```

Comment:
Change to:  For insight into what is happening under the hood you can use `-x` or `--print-commands` to see how the processes are started